networkx/algorithms/traversal/reachability.py

```diff
@@ -0,0 +1,214 @@
+"""Algorithms related to reachability problems"""
+import networkx as nx
+
+__all__ = [
+    "descendants",
+    "ancestors",
+    "transitive_closure",
+    "descendants_at_distance",
```

Comment:
Major nit, but since we're starting fresh here it might be nice to list them alphabetically :)

Reply:
Agree :sweat_smile: